prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -945,13 +995,66 @@ class GetMultipleSettings : SettingCommand(
         }
 
         // get senders and receivers per org
-        val deepOrganizations = organizations.map { org ->
+        return organizations.map { org ->
             val sendersJson = getMany(environment, accessToken, SettingType.SENDER, org.name)
             val orgSenders = jsonMapper.readValue(sendersJson, Array<Sender>::class.java).map { it.makeCopy() }
             val receiversJson = getMany(environment, accessToken, SettingType.RECEIVER, org.name)
             val orgReceivers = jsonMapper.readValue(receiversJson, Array<ReceiverAPI>::class.java).map { Receiver(it) }
             DeepOrganization(org, orgSenders, orgReceivers)
         }
-        return yamlMapper.writeValueAsString(deepOrganizations)
+    }
+
+    private fun loadSettings(settings: List<DeepOrganization>) {
+        if (settings.isNotEmpty()) {
+            // Change transports to SFTP
+            val modifiedOrgs = settings.map { org ->
+                val modifiedReceivers = org.receivers.map {
+                    Receiver(
+                        it.name,
+                        it.organizationName,
+                        it.topic,
+                        it.customerStatus,
+                        it.translation,
+                        it.jurisdictionalFilter,
+                        it.qualityFilter,
+                        it.routingFilter,
+                        it.processingModeFilter,
+                        it.reverseTheQualityFilter,
+                        it.conditionFilter,
+                        it.deidentify,
+                        it.deidentifiedValue,
+                        it.timing,
+                        it.description,
+                        localTransport,
+                        it.externalName,
+                        it.timeZone,
+                        it.dateTimeFormat
+                    )
+                }
+                DeepOrganization(org, org.senders, modifiedReceivers)
+            }
+            val differences = diffAll(modifiedOrgs, Environment.LOCAL, dummyAccessToken)
+            if (differences.isNotEmpty()) {
+                echoDiff(differences)
+                confirm()
+            }
+            val output = putAll(modifiedOrgs, Environment.LOCAL, dummyAccessToken)
+            echo("Loaded settings to local DB:")
+            echo("${output.joinToString("\n")}\n")
+            if (appendToOrgs) {
+                echo("Adding settings that were loaded to the local database to the organizations.yml file")
+                appendToOrgs(yamlMapper.writeValueAsString(modifiedOrgs))
+            }
+        } else {
+            echo("No settings fitting your parameters were returned.")
+        }
+    }
+
+    private fun appendToOrgs(output: String) {
+        if (output.startsWith("---")) {
```

Comment:
what does the `---` prefix signify?

Reply:
I'm not actually super sure what that is about, but it's at the start of the yml files and since we're appending, we don't want it.